src/layouts/MyLayout.vue

```diff
@@ -4,110 +4,116 @@
       elevated
     >
       <q-toolbar>
+        <q-toolbar-title>
+          <font
+            face="impact"
+            size="6"
+          >
+            ICEMEME
+          </font>
+        </q-toolbar-title>
```

Comment:
- don't use a `font` element ([it's obsolete](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/font)), instead define a class on the `q-toolbar-title` that sets the style
- wrap the `q-toolbar-title` in a `router-link` element linking to the home page